kcards/templates/join.html

```diff
@@ -3,9 +3,13 @@
 {% block title %} | Join Room{% endblock %}
 
 {% block body %}
-<p>What is your name?</p>
-<form method="post">
-  <input type="text" name="name" placeholder="Firstname Lastname"></input>
-  <input type="submit" name="join" value="Join Room"></input>
-</form>
+    <p>What is your name?</p>
+    <form method="post">
+        <input type="text" name="name" placeholder="Firstname Lastname"></input>
+        <input type="submit" name="join" value="Join Room"></input>
+    </form>
 {% endblock%}
+
+{% block footer %}
+    <div class="footer">{{ code }}</div>
```

Comment:
We need the room code to appear somewhere on the page for testing. I think ultimately we'll want the room code on the page somewhere for the user.